lib/src/buttons/help_button.dart

```diff
@@ -0,0 +1,208 @@
+import 'package:flutter/cupertino.dart'
+    show CupertinoColors, CupertinoDynamicColor, CupertinoIcons;
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+import '../../macos_ui.dart';
+
+/// A macOS-style button.
+class HelpButton extends StatefulWidget {
+  const HelpButton({
```

Comment:
Documentate the constructor. It must say that `pressedOpacity`, if non-null, must be in the range if 0.0 to 1.0